### PR TITLE
Emphasise unique capabilities, update Talk and add NWU case study

### PR DESCRIPTION
**Pull request by jospoortvliet**
This has a bunch of changes.

# Self-hosted
First, it updates various pages with textual updates to make clear that our main 'feature' is self-hosting/meta-data protection. Affected pages include Talk, About and Home page.
On the Groupware page I tried to make the integration with Files a bit more obvious.

# Talk updates
The Talk page got further updates, with a nice pic of the new chat app and updated pricing.

# NWU casestudy
I also added the NWU case study to the Education and Collabora Online page.

screenshots:

NWU case study as on Collabora Online page (same look on Education page):
screenshot_20180615_224924

Updated text section on top and key feature list on Talk page.
screenshot_20180615_225126

Image besides 'get mobile clients' on Talk page (properly responsive!)
screenshot_20180615_225228

New 'why Nextcloud' texts on front page.
screenshot_20180615_225339

### CODE REVIEW

**Comment by jasonbayton** [comment]
Suggestions (take or leave): 

Why Nextcloud?: Control: 

"Protect, control and monitor data and communication _across_ your company"

Does that sound better?

Why Nextcloud?: Productivity:

"Enable _productivity_ across any platform, _whether in the office or on the road_, to share, collaborate and communicate _across organizational_ boundaries"

Why Nextcloud?: Community:
"_entirely_ .. or _fully_ open-source community development model"

New image on the app page:
Can I boycott this merge just because you've used an iPhone rather than Android? :stuck_out_tongue: 

Case study: 
"Get the free case study:" - maybe bold or subtitle this?
Can you increase the size of the preview, or do something to bring the top of the image close in line with the top of the heading? Might look better.

Talk page:
metadata or meta-data or meta data? Make this consistent across the content.

_can still see who you've communicated with_ or _who you communicate with

_This ensures you stay in complete control of communications_

HTH!

File: page-homepage.php [hunk 1]
```diff
@@ -36,17 +36,17 @@
...
+				<p class="section--paragraph revealOnScroll"><?php echo $l->t('Protect, control and monitor data and communication in your company. Guarantee compliance with business and legal requirements. Keep your data on servers you own, at all times. Nothing leaks, not even meta-data.');?></p>
 			</div>
 			<div class="highlights_align">
 				<div class="flexibility-icon" data-animation-path="img/home/flexibility/" data-anim-loop="false" data-name="flexibility"></div>
 				<h4 class="revealOnScroll"><?php echo $l->t('Productivity');?></h4>
-				<p class="section--paragraph revealOnScroll"><?php echo $l->t('Make data available on any platform to your teams. Let them share, collaborate and communicate within and across organizational boundaries.');?></p>
+				<p class="section--paragraph revealOnScroll"><?php echo $l->t('Let your teams access data wherever it is stored. Enable their work on any platform, mobile or at the office, to share, collaborate and communicate within and across organizational boundaries. ');?></p>
```
Comment (tflidd) [inline page-homepage.php]: the "wherever" is a bit strange here. A few lines above, you point out the advantage of self-hosted solutions. Do you refer to external storage that can be included?

- not all storage types are possible, gdrive or dropbox app are currently not working
- it's not really clear what is meant by wherever, if it is on the server or perhaps the client device which can be wherever. If you want to refer to different external storage, I'd rather say "Let your team access data from different/various storage back-ends". And link to the documentation with external storage.

Just by reading, perhaps restructure the sentences:

> Let your teams work on any platform, mobile.... Nextcloud consolidates different storage back-ends seamlessly.

**Comment by tflidd** [comment]
North-West University probably needs some specification since there are several of them https://en.wikipedia.org/wiki/Northwest_University_(disambiguation)

File: page-homepage.php (hunk 1 above)
Comment (jospoortvliet) [reply page-homepage.php]: I did mean to talk about external storage, yeah. GDrive or Dropbox are of course not relevant for companies, they care mostly about Windows Network Drive, NFS and a bit of SharePoint...

But yeah, this was very hard to put in, I used a modification on your wording.

**Pull request merged**